src/ansys/fluent/core/solver/flobject.py

```diff
@@ -803,16 +810,38 @@ class FileName(Base):
     pass
 
 
+def expand_api_file_argument(command_name, value, kwargs):
+    """Expand API file argument."""
+    if kwargs.get("file_type") == "case-data" or command_name in [
+        "read_case_data",
+        "write_case_data",
+    ]:
+        data_file = value.replace(".cas.h5", ".dat.h5")
```

Comment:
Are't we supporting .cas and .cas.gz also? Therefore, we can omit .h5 here.

Reply:
Yes. Done. Thank you.